website/content/api-docs/secret/pki.mdx

```diff
@@ -1963,6 +1964,52 @@ $ curl \
 }
 ```
 
+### Revoke Issuer
+
+This endpoint allows an operator to revoke an issuer certificate, marking it
+unable to issue new certificates and adding it to other issuers' CRLs, if they
+have signed this issuer's certificate.
```

Comment:
We should probably call out CRL will be updated if the signing cert is also in the same mount?

Reply:
It'll unconditionally be rebuilt (but only added if it exists), but yeah, good idea.